packages/hoppscotch-common/src/helpers/teams/TeamCollectionAdapter.ts

```diff
@@ -1080,7 +1085,7 @@ export default class NewTeamCollectionAdapter {
           authType: "inherit",
           authActive: true,
         }
-        auth.parentID = [...path.slice(0, i + 1)].join("/")
+        auth.parentID = path.slice(0, i + 1).join("/")
```

Comment:
Since this is consumed at multiple places, can we set it at the top and refer it everywhere else? Please ensure there aren't any side effects.